WooCommerce/src/androidTest/kotlin/com/woocommerce/android/ui/main/ProductsUITest.kt

```diff
@@ -0,0 +1,71 @@
+package com.woocommerce.android.ui.main
+
+import androidx.test.rule.ActivityTestRule
+import com.woocommerce.android.BuildConfig
+import com.woocommerce.android.helpers.InitializationRule
+import com.woocommerce.android.helpers.TestBase
+import com.woocommerce.android.screenshots.TabNavComponent
+import com.woocommerce.android.screenshots.login.WelcomeScreen
+import com.woocommerce.android.screenshots.products.ProductListScreen
+import com.woocommerce.android.screenshots.util.ProductData
+import com.woocommerce.android.screenshots.util.MocksReader
+import com.woocommerce.android.screenshots.util.iterator
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@HiltAndroidTest
+class ProductsUITest : TestBase() {
+    @get:Rule(order = 0)
+    val rule = HiltAndroidRule(this)
+
+    @get:Rule(order = 1)
+    val initRule = InitializationRule()
+
+    @get:Rule(order = 3)
```

Comment:
Is there a reason this is not `order = 2`?

```suggestion
    @get:Rule(order = 2)
```

Reply:
Thanks for noticing! Changing it to `2` does not break the tests, so 'fixed' it in 2309053bf7af73986c5569b00400bb930ffe2d07 for both files.